server/routerlicious/packages/routerlicious/config/config.json

```diff
@@ -139,7 +139,13 @@
         }
     },
     "scribe": {
-        "kafkaClientId": "scribe"
+        "kafkaClientId": "scribe",
+        "checkpointHeuristics": {
```

Comment:
Do we also need to add these new settings to the fluid-configmap.yaml that defines the config.json used in Kubernetes?

https://github.com/microsoft/FluidFramework/blob/main/server/routerlicious/kubernetes/routerlicious/templates/fluid-configmap.yaml

Reply:
Added the new values